### ISSUE TITLE
Template not treated as HTML in VSCode

### ISSUE THREAD

**Issue by gianLuigiL** [vuejs/language-tools#4016]
Updating to latest official vue extension stopped vscode from treating the template section in Vue SFC as HTML.
Example: auto completion of quotes after the equal sign stopped working.

Adding `:test=` will not trigger automatic quotes insertion anymore. In common HTML files this works as expected.

**Comment by gianLuigiL** [comment]
[screencast-bpconcjcammlapcogcnnelfmaeghhagj-2024.03.07-09_23_08.webm](https://github.com/vuejs/language-tools/assets/41172153/95ce4b75-9fac-4d51-a997-edcff50b2358)
Yes, thanks.

In this video notice how in the previous version:
- The closing angle bracket on the opening `<Accordion>` tag triggered autocompletion for the closing `</Accordion>` tag.
- The attribute `hello=` triggered automatically adding the quotes. They are not inserted automatically anymore. 
It appears all behaviours defined for HTML in vscode stopped applying to the template section of SFC

**Comment by so1ve** [comment]
`Accordion` doesn't have correct highlight so it seemed that the plugin isn't loaded correctly.

**Comment by gianLuigiL** [comment]
Is there something actionable i can do to help?

**Comment by so1ve** [comment]
Sorry I'm not sure how to deal with this part, we may wait for @johnsoncodehk's input

**Comment by hermesjpappas** [comment]
@gianLuigiL Any luck solving this?

**Comment by gianLuigiL** [comment]
> @gianLuigiL Can you reproduce this problem with v2.0.12?

Yes still stuck on the same behaviour. No auto closing tags and no auto closing quotes.
I'm running 1.8.27 for now.

**Comment by johnsoncodehk** [comment]
Hybrid Mode is always enabled after 3.0. If you still have problems, please open a new issue.
